omero/sysadmins/limitations.rst

```diff
@@ -62,6 +62,7 @@ The OMERO pyramid generation process should be considered as deprecated and inst
 that users avoid these issues by converting
 their data to `pyramidal OME-TIFF <https://www.openmicroscopy.org/2018/11/29/ometiffpyramid.html>`_
 files before importing into OMERO. A number of suitable tools are available such as
+`NGFF-Converter <https://www.glencoesoftware.com/products/ngff-converter/>`_,
```

Comment:
I think it will be preferable to point to https://ngff.openmicroscopy.org/tools/index.html since the list will grow and we do not want to maintain 2 lists

Reply:
> I think it will be preferable to point to https://ngff.openmicroscopy.org/tools/index.html since the list will grow and we do not want to maintain 2 lists

These are 2 different lists. The list in this PR (in the paragraph of "Limitations") is dealing with conversion tools to ome-tiff.
The https://ngff.openmicroscopy.org/tools/index.html is dealing with conversion to ngff (which we do not recommend afaik for this workflow) - e.g. QuPath, the Fiji tools etc. is missing from https://ngff.openmicroscopy.org/tools/index.html (for the reasons mentioned.

Reply:
understood